src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -153,61 +154,84 @@ const FormattedReport = ({ summaries, weekIndex }) => {
+        {getTotalValidWeeklySummaries(summary)}
+        {hoursLogged >= summary.weeklycommittedHours && (
+          <p>
+            <b>Hours logged:</b> {hoursLogged.toFixed(2)} / {summary.weeklycommittedHours}
+          </p>
+        )}
+        {hoursLogged < summary.weeklycommittedHours && (
+          <p style={{ color: 'red' }}>
+            <b>Hours logged:</b> {hoursLogged.toFixed(2)} / {summary.weeklycommittedHours}
+          </p>
+        )}
+        {getWeeklySummaryMessage(summary)}
+      </div>
+    );
+  };
+
   return (
     <>
-      {alphabetize(summaries).map((summary, index) => {
-        const hoursLogged = (summary.totalSeconds[weekIndex] || 0) / 3600;
-        const googleDocLink = getGoogleDocLink(summary);
-        const [isBioPosted, setIsBioPosted] = useState(summary.bioPosted);
-        return (
-          <div
-            style={{ padding: '20px 0', marginTop: '5px', borderBottom: '1px solid #DEE2E6' }}
-            key={'summary-' + index}
-          >
-            <div>
-              <b>Name: </b>
-              <Link to={`/userProfile/${summary._id}`} title="View Profile">
-                {summary.firstName} {summary.lastName}
-              </Link>
-
-              <span onClick={() => handleGoogleDocClick(googleDocLink)}>
-                <img className="google-doc-icon" src={google_doc_icon} alt="google_doc" />
-              </span>
-            </div>
-            <div>
-              {' '}
-              <b>Media URL:</b> {getMediaUrlLink(summary)}
-            </div>
-            <div>
-              <div className="bio-toggle">
-                <b>Bio announcement:</b>
-              </div>
-              <div className="bio-toggle">
-                <ToggleSwitch
-                  switchType="bio"
-                  state={isBioPosted ? false : true}
-                  handleUserProfile={() => {
-                    handleChangeBioPosted(summary._id, isBioPosted);
-                    setIsBioPosted(!isBioPosted);
-                  }}
-                />
-              </div>
-            </div>
-            {getTotalValidWeeklySummaries(summary)}
-            {hoursLogged >= summary.weeklycommittedHours && (
-              <p>
-                <b>Hours logged:</b> {hoursLogged.toFixed(2)} / {summary.weeklycommittedHours}
-              </p>
-            )}
-            {hoursLogged < summary.weeklycommittedHours && (
-              <p style={{ color: 'red' }}>
-                <b>Hours logged:</b> {hoursLogged.toFixed(2)} / {summary.weeklycommittedHours}
-              </p>
-            )}
-            {getWeeklySummaryMessage(summary)}
-          </div>
-        );
-      })}
+      {bioCanEdit
+        ? alphabetize(summaries).map((summary, index) => {
+            return summaryCard(summary, index, bioSwitch);
+          })
+        : alphabetize(summaries).map((summary, index) => {
+            return summaryCard(summary, index, bioLabel);
+          })}
```

Comment:
If you define `bioFunction` within `summaryCard`, this can be: `{alphabetize(summaries).map((summary, index) => summaryCard(summary, index))}`

Reply:
1) Line 186: `const bioFunction = bioCanEdit ? bioSwitch : bioLabel;`.
2) Line 187: `const summaryCard = (summary, index) => {`.
3) Lines from 228 to 234: `{alphabetize(summaries).map((summary, index) => summaryCard(summary, index))}`.